Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,84 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {

```

Comment:
Code is too big for a single function.
Ideally function should not have more than 20-30 lines. If it has more line than that, break it into multiple functions.
Code in different if statements could be separate functions, so that it becomes easy to read.
Add comments of what your logic is as it is very difficult to drive it from reading code.